### PR TITLE
Changes in html, css, and javaScript

### PR DESCRIPTION
**Pull request by Swiftyfast**
In html, changed the top title.

In css, changed the background color of text in InsideHouse

In js, worked on the second kiva api and changed it so council district and trash day are options to display.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -52,4 +52,7 @@ body {
   height: 100%;
   color: #FFFFCC;
   background-color:#000080;
+}
+.displayText{
```
Comment (sandipsubedi) [inline src/index.css]: classes are normally written as `display-text`

File: src/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 import axios from 'axios';
 
-var myObject = {"KIVA": "4"};
+var myObject = {"KIVA": "4", "cityCouncilDistrict": "", "trashPickUp": ""};
```
Comment (sandipsubedi) [inline src/index.js]: `let`

File: src/index.js [hunk 3]
```diff
@@ -37,15 +37,23 @@ class FrontDoor extends React.Component{
         event.preventDefault();
         axios.get(sentAddress).then(function(response){
             var myResponse = response.data;
```
Comment (sandipsubedi) [inline src/index.js]: `let`

File: src/index.js [hunk 4]
```diff
@@ -37,15 +37,23 @@ class FrontDoor extends React.Component{
...
-            console.log("this is working");
             myObject.KIVA = myResponse.data.city_id;
-            console.log(myObject.KIVA);
+            myObject.cityCouncilDistrict =  myResponse.data.city_council_district;
+            var myTestKiva = "http://maps.kcmo.org/kcgis/rest/services/external/Tables/MapServer/2/" + myResponse.data.city_id + "?f=json&pretty=true";
```
Comment (sandipsubedi) [inline src/index.js]: `let`

File: src/index.js [hunk 5]
```diff
@@ -37,15 +37,23 @@ class FrontDoor extends React.Component{
...
+            axios.get(myTestKiva).then(function(response){
+                //console.log("this part worked");
+                var mySubResponse = response.data;
```
Comment (sandipsubedi) [inline src/index.js]: `let`

File: src/index.js [hunk 6]
```diff
@@ -97,12 +105,12 @@ class InsideHouse extends React.Component{
         return(
             <div>
             <div className="menuBar">
-                <div><label><input type="radio" name="myRadio" value="rad1" onChange={this.changeStuff} checked={this.state.doorway === 'rad1'} /><p1>Kiva Pin (not working)</p1></label></div>
-                <div><label><input type="radio" name="myRadio" value="rad2" onChange={this.changeStuff} checked={this.state.doorway === 'rad2'}/><p1>Radio2</p1></label></div>
-                <div><label><input type="radio" name="myRadio" value="rad3" onChange={this.changeStuff} checked={this.state.doorway === 'rad3'}/><p1>Radio3</p1></label></div>
+                <div><label><input type="radio" name="myRadio" value="rad1" onChange={this.changeStuff} checked={this.state.doorway === 'rad1'} /><p1>Recycling Information</p1></label></div>
+                <div><label><input type="radio" name="myRadio" value="rad2" onChange={this.changeStuff} checked={this.state.doorway === 'rad2'}/><p1>City Council District</p1></label></div>
+                <div><label><input type="radio" name="myRadio" value="rad3" onChange={this.changeStuff} checked={this.state.doorway === 'rad3'}/><p1>Trash Day</p1></label></div>
```
Comment (jzeilmann93) [inline src/index.js]: Use p tag instead of p1

**Pull request merged**